base/src/main/java/com/smartdevicelink/managers/file/BaseFileManager.java

```diff
@@ -431,40 +463,60 @@ public void uploadArtworks(List<SdlArtwork> files, final MultipleFileCompletionL
     }
 
     /**
-     * Check if an SdlFile has been uploaded to core
+     * Checks if an artwork needs to be uploaded to Core. The artwork should not be sent to Core if
+     * the artwork is already on Core or if the artwork is not on Core after the maximum number of
+     * repeated upload attempts has been reached.
      *
-     * @param file SdlFile
-     * @return boolean that tells whether file has been uploaded to core (true) or not (false)
+     * @param file                   The file to be uploaded to Core
```

Comment:
Do you mean to have all this extra space between file and its description in the JavaDoscs? I think the autoFormatter does this and I noticed this throughout this PR.

Reply:
Yes, the auto formatter in AS does that. That should be consistent in all classes in the project. When we did the refactoring in 5.0, we did let the formatter format all existing code in the library. So all classes should have the extra spaces to align.